spec-0001.md

```diff
@@ -165,9 +193,11 @@ functions:
 ...
 ```
 
-Ultimately, we hope that lazy importing will become part of Python itself.
+Ultimately, we hope that lazy importing will become part of Python itself, but the developers have indicated that this is highly unlikely [^cannon].
 In the mean time, we now have the necessary mechanisms to implement it ourselves.
 
+[^cannon]: Cannon B., private communications, 7 January 2021.
```

Comment:
Maybe "personal communication"?  It is probably fine, but "private" sort of conveys it was supposed to be secret (to me at least).  Feel free to ignore.

Reply:
Woops, thanks for catching that!  Fixed.